lib/ravelin.rb

```diff
@@ -1,5 +1,39 @@
-require "ravelin/version"
+require 'date'
```

Comment:
I don't have a good reason other that it felt okay to require the library dependancies and library files in the same place. I also followed the style of a couple of other Ruby API wrapper gems.

Would you suggest requiring them in the exact files where they are needed?